src/main/kotlin/com/manimdsl/linearrepresentation/LinearRepresentation.kt

```diff
@@ -205,12 +95,16 @@ data class StackPopObject(
     override fun toPython(): List<String> {
         val instruction = getInstructionString("animation", true)
         return listOf(
-            "[$instruction for animation in $dataStructureIdentifier.pop(${shape.ident}, fade_out=${(!insideMethodCall).toString()
-                .capitalize()})]"
+            "[$instruction for animation in $dataStructureIdentifier.pop($ident, fade_out=${
+            (!insideMethodCall).toString()
+                .capitalize()
+            })]"
         )
     }
 }
 
+/** Arrays **/
```

Comment:
Can data structures be separated out into their own files to promote extensibility?